PWGHF/TableProducer/candidateCreator2Prong.cxx

```diff
@@ -74,24 +88,34 @@ struct HfCandidateCreator2Prong {
   OutputObj<TH1F> hCovSVZZ{TH1F("hCovSVZZ", "2-prong candidates;ZZ element of cov. matrix of sec. vtx. position (cm^{2});entries", 100, 0., 0.2)};
   OutputObj<TH2F> hDcaXYProngs{TH2F("hDcaXYProngs", "DCAxy of 2-prong candidates;#it{p}_{T} (GeV/#it{c};#it{d}_{xy}) (#mum);entries", 100, 0., 20., 200, -500., 500.)};
   OutputObj<TH2F> hDcaZProngs{TH2F("hDcaZProngs", "DCAz of 2-prong candidates;#it{p}_{T} (GeV/#it{c};#it{d}_{z}) (#mum);entries", 100, 0., 20., 200, -500., 500.)};
+  OutputObj<TH1F> hVertexerType{TH1F("hVertexerType", "Use KF or DCAFitterN;Vertexer type;entries", 2, -0.5, 1.5)}; // See o2::aod::hf_cand::VertexerType
 
   void init(InitContext const&)
   {
-    if (doprocessPvRefit && doprocessNoPvRefit) {
-      LOGP(fatal, "Only one process function between processPvRefit and processNoPvRefit can be enabled at a time.");
+    std::array<bool, 2> doprocessDF{doprocessPvRefitWithDCAFitterN, doprocessNoPvRefitWithDCAFitterN};
```

Comment:
since you kindly do in the `taskD0`, what about adding also here a check that throws a `FATAL` in case no `process` functions are activated?

Reply:
Hi @mfaggin , I am not sure I understand your suggestion. The current code in the PR already does throw a fatal if no process functions are activated. What would you do differently?

Reply:
As far as I understand, what is present now throws a `FATAL` only if more than 1 `process` functions are active.

In the `taskD0.cxx` instead (see here https://github.com/AliceO2Group/O2Physics/pull/3046/files#diff-4a821720ef7d4690565c521e69f02edbf6192c257697a2f4afb8898b98d7d714R128) there is also the check whether at least 1 process function is active or not. I'm just suggesting do add it as well also here.

Not a major thing, though.

Reply:
As far as I can see, the condition for throwing a fatal is ` != 1` which also includes `0`.

Reply:
The case of `taskD0` is more complicated because it is legit to enable one process function for data and one for MC independently. So the first check tests that some process function is enabled and the second check tests the right combination. For the other workflows we want to allow exactly one process function to be enabled, hence the condition `!= 1` for the fatal.

Reply:
you are right, sorry, I got confused by the message